homeassistant/components/solarlog/const.py

```diff
@@ -145,8 +144,7 @@ class SolarLogSensorEntityDescription(SensorEntityDescription):
         name="consumption total",
         native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
         device_class=DEVICE_CLASS_ENERGY,
-        state_class=STATE_CLASS_MEASUREMENT,
-        last_reset=dt.utc_from_timestamp(0),
+        state_class=STATE_CLASS_TOTAL,
```

Comment:
Assuming the total consumption sensor never resets, the state class should be set to `total` without `last_reset`